rusoto/services/sts/src/custom/web_identity.rs

```diff
@@ -30,23 +30,21 @@ pub struct WebIdentityProvider {
     /// associated with the user who is using your application. That way, the temporary security credentials
     /// that your application will use are associated with that user. This session name is included as part
     /// of the ARN and assumed role ID in the AssumedRoleUser response element.
-    pub role_session_name: Variable<String, CredentialsError>,
+    pub role_session_name: Variable<Option<String>, CredentialsError>,
 }
 
 impl WebIdentityProvider {
     /// Create new WebIdentityProvider by explicitly passing its configuration.
-    pub fn new<A, B, C>(web_identity_token: A, role_arn: B, role_session_name: Option<C>) -> Self
+    pub fn new<A, B, C>(web_identity_token: A, role_arn: B, role_session_name: C) -> Self
```

Comment:
Now, one has to always pass a Variable for `role_session_name`, while with the former approach one could provide `None` here. I propose we change this to:
```rust
pub fn new<A, B, C>(web_identity_token: A, role_arn: B, role_session_name: Option<C>) -> Self
```